### ISSUE TITLE
Changes to the code

### ISSUE THREAD

**Issue by admiralmaggie** [RolandJuno/arduino2atari1020#1]
I was able to get this working. Let me know if you want me to upload a new version.

**Comment by RolandJuno** [comment]
Sure, that would be great!

**Comment by sprive** [comment]
@admiralmaggie did you ever submit that Pull request (upload)?

**Comment by admiralmaggie** [comment]
@sprive No... Completely forgot. Let me check to see if I can find it...

**Comment by RolandJuno** [comment]
@admiralmaggie Did you happen to locate the changes you made to the code to get printing to Atari printers from an Arduino working?

**Comment by admiralmaggie** [comment]
@RolandJuno Found it. How do you want me to submit it? Pull and push?

**Comment by admiralmaggie** [comment]
@RolandJuno I created a pull request and uploaded the code under atari1020v2.ino

**Comment by sprive** [comment]
NICE - TY both! If (anyone) a video link can be posted, that would be sweet. These things didn't break over time like the LQ printers; I'll have to dig one up.

**Comment by zjpiazza** [comment]
I was actually looking to use an Atari 1020 in a project and would love to get some more information. Firstly, I'm a little confused...is there a specific reason why an Arduino is needed? Why couldn't you just use something like 
SIO2PC adapter?

**Comment by admiralmaggie** [comment]
@zjpiazza I don't think you need the Arduino perse but I didn't have a SIO2PC adapter.:)
